packages/compass-connect/src/stores/index.js

```diff
@@ -1080,6 +1038,7 @@ const Store = Reflux.createStore({
 
       this._onConnectSuccess(connectedDataService, connectionInfo);
     } catch (error) {
+      trackConnectionFailedEvent(connectionInfo, error);
```

Comment:
probably not an issue since this code will go away but there is no guarantee for `connectionInfo` to be defined at this point